pkg/keymanagementprovider/azurekeyvault/provider.go

```diff
@@ -64,15 +65,15 @@ type AKVKeyManagementProviderConfig struct {
 }
 
 type akvKMProvider struct {
-	provider     string
-	vaultURI     string
-	tenantID     string
-	clientID     string
-	cloudName    string
-	certificates []types.KeyVaultValue
-	keys         []types.KeyVaultValue
-	cloudEnv     *azure.Environment
-	kvClient     *kv.BaseClient
+	provider       string
+	vaultURI       string
+	tenantID       string
+	clientID       string
+	cloudName      string
```

Comment:
is `cloudName` used anywhere still? can we remove it?

Reply:
Nope, I'll remove it.